src/libraries/System.Globalization/System.Globalization.sln

```diff
@@ -26,6 +26,9 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{2E666815-2EDB-464B-9DF6-380BF4789AD4}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Globalization.Nls.Tests", "tests\NlsTests\System.Globalization.Nls.Tests.csproj", "{6B71E284-DA57-4657-9E08-A02BAFB877CC}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
Is this needed?

Reply:
I believe so. All other test projects have this same annotation such that they take a build ordering dependency on the src project.

Reply:
I see, yeah makes sense. Do we need to do the same for System.Runtime.Nls.Tests? in System.Runtime.sln?

Reply:
Ah, yeah, good catch!